Kajabi/sage-lib
# [Sage-981] LabelGroup: Spacing adjustment

## Description

Remove margin on label groups in favor of gap.

## Screenshots

|  Before  |  After  |
|--------|--------|
| ![before-label-group](https://user-images.githubusercontent.com/24237393/141188383-33bebe42-309b-49b1-96a2-72ad499f94b2.png) | ![Screen Shot 2021-11-11 at 8 44 09 AM](https://user-images.githubusercontent.com/24237393/141317573-d20f3e0a-5ed4-4cd1-bdc9-20122f531eb3.png)  |

## Testing in `sage-lib`
1. Navigate to http://localhost:4000/pages/component/label
2. Verify gap is spacing the individual labels correctly


## Testing in `kajabi-products`
1. (**MEDIUM**) Spacing update in label groups. Verify that the UI is still rendering correct
   - [ ] Various areas of Super Admin
   - [ ] Grants / Offer List Item

Individual files with SageLabelGroup:
* `app/views/super_admin/audit_control_logs/_active_filters.html.erb`
* `app/views/super_admin/audit_control_logs/index.html.erb`
* `app/views/super_admin/koms/site_authorizations/_control_defaults_alert.html.erb`
* `app/views/super_admin/koms/site_authorizations/_status_filter.html.erb`
* `app/views/super_admin/koms/site_authorizations/_site_control_labels.html.erb`
* `app/views/super_admin/audit_risk_logs/_active_filters.html.erb`
* `app/views/super_admin/audit_risk_logs/index.html.erb`
* `app/views/admin/grants/_offer_list_item.html.erb`

## Related
Closes #981 
Related `kajabi-products` PR: kajabi/kajabi-products#21372
